backend/src/Github/Agent/GithubCodeReviewerAgent.php

```diff
@@ -120,6 +122,13 @@ public function processTask(
         }
 
         if (!$completion->choices[0]->message->content) {
+            if ($toolsCalled) {
+                $reqMessages[] = new AgentMessage(
+                    'Max tool calls reached. ' .
+                    'Provide a remaining brief summary without repeating feedback already provided by tools.',
+                    AgentMessageRole::USER
```

Comment:
Ensure that the line `if ($toolsCalled) {` truly reflects the intended behavior here, especially in the context of determining when to send a brief summary. The current logic might not account for cases where tool calls were attempted but not successfully executed due to rate limits or other errors.